tracee-ebpf/tracee/tracee.go

```diff
@@ -902,6 +940,33 @@ func (t *Tracee) populateBPFMaps() error {
 		}
 	}
 
+	err = t.populateProcessTreeMap(t.config.Filter.ProcessTreeFilter.PID)
```

Comment:
This map is populated before the bpf programs are attached.
If processes are added/remved after this point, they will not be added/removed from the map.
We should probably do another iteration after the bpf programs are attached, and check if no changes occured to avoid race conditions